lib/inspec/backend.rb

```diff
@@ -77,6 +77,14 @@ def self.create(config) # rubocop:disable Metrics/AbcSize
           connection
         end
 
+        def method_missing(id, *args, &blk)
```

Comment:
I feel that at this point we should be careful to heavily explain what is happening when we add new elements of metaprogramming. What context is this being called in? Explain it to most novice new developer on the team, but also to the potential contributor, in comments. Right now it is fresh in your mind and obvious; it will never be easier.

I can follow this one, but I have the tribal knowledge that requiring a resource has the side effect registering it in the registry.

Reply:
Thanks!